Task 7/Task 7.3/storage.js

```diff
@@ -0,0 +1,82 @@
+export { Storage }
+
+class Storage {
+
+    _storage = [];
+    _idCount = 1;
+
+    add(obj) {
+        this.checkIncomingElement(obj);
+        this.getObjectID(obj);
+        this._storage.push(obj);
+    };
+
+    getById(id) {
+        this.checkTypeOfID(id);
+        let resultObj = this._storage.find(item => item.id === id);
+        if (resultObj === undefined) {
+            return null;
```

Comment:
better leave it as undefined. It's not a C#, there is a difference between `null` and `undefined`, and it is quite literal. If an element doesn't exist, it is _not defined_, if it doesn't exist but explicitly (a place is reserved, but no object) - it's `null`

Reply:
There was special condition in the task, that if object is not found by id, you should return null. Maybe it was meant "null"???